test.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+source settings.sh
+DEVICE=ltsptest01
+
+help() {
+    cat <<-_EOF
+    
+test.sh
+This script creates virtual network adapter which can be used for bridging
+
+Usage:
+$0 [options]
+
+Options:
+    start   This option will create virtual adpater
+    stop    This option will destory virtual adpater
+    
+_EOF
+}
+
+if [ $UID != 0 ]; then
+	echo "error: use sudo or run script as root user"
+	exit
+fi
+
+if [ "$#" == 0 ]; then
+    echo "error: No arguments provided"
+    help
+    exit
+fi
+
+case "$1" in
+    "start" )
+        echo "creating virtual interface.."
+        modprobe dummy
+        ip link add ${DEVICE} type dummy
+        ip link set ${DEVICE} up
+        ip addr add ${NETWORK}.100/24 brd + dev ${DEVICE}
+        echo "INTERFACE=\"${DEVICE}\"" >> settings.sh 
```

Comment:
Before adding something automatically to a configuration file, it is a good practice to try to delete it first, in order to make the operation _idempotent_.

Reply:
very nice!